src/apps/favorites-list/favorites-list.test.ts

```diff
@@ -45,31 +51,75 @@ describe("Favorites list", () => {
     cy.visit("/iframe.html?id=apps-favorite-list--primary&viewMode=story");
   });
 
+  // Test that the list of favorites is displayed with materials correctly
   it("Favorites list basics", () => {
     // Wait for element not in skeleton screen to prevent testing prematurely.
     cy.get(".cover").should("be.visible");
 
-    // 2.a. Header "Favorites"
+    // Header has "Favorites"
     cy.get(".content-list-page").find("h1").should("have.text", "Favorites");
-    // Number of materials on list
+
+    // Sub header shows correct number of materials
```

Comment:
And this one would be something like `it("Shows the correct number of materials in sub-header")`